docs/docs/reference/release-notes/v2.30/index.md

```diff
@@ -0,0 +1,189 @@
+---
+date: "2021-01-05"
+version: "2.30.0"
+---
+
+# [v2.30](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.30.0-next.0...gatsby@2.30.0) (January 2021 #1)
+
+---
+
+Welcome to `gatsby@2.30.0` release (January 2021 #1)
+
+Key highlights of this release:
+
+- [Query on Demand and Lazy Images: Generally available](#query-on-demand-and-lazy-images-generally-available) - improves `gatsby develop` bootup time
+- [Server Side Rendering (SSR) in development](#server-side-rendering-ssr-in-development) — helps you find and fix many build errors in development. We're starting a partial release of this feature to 5% of users.
+- [Image transformations up to 30% faster](#gatsby-plugin-sharp2120)
+- [gatsby-plugin-sass v3](#gatsby-plugin-sass300) - use the latest `sass-loader` and `Dart Sass` by default
+
+And several impactful updates in the new [`gatsby-plugin-image`](#gatsby-plugin-image050-beta) (beta):
+
+- [AVIF support](#avif-support) - enjoy next-gen image format
+- [Support remote static images](#support-static-image-urls-experimental) - download, transform and optimize remote images with a single line
+- [New defaults](#new-defaults) - less configuration for most common image setups
```

Comment:
I don't see "New defaults" section in content?

Reply:
True, looks like Matt decided not to include this part

Reply:
@ascorbic I was thinking about including a note about this PR: https://github.com/gatsbyjs/gatsby/pull/28662 there Will remove this "New defaults" section for now.

Reply:
Oh, I didn't realise that was in this release!